avocado/core/remoter.py

```diff
@@ -194,6 +216,7 @@ def __init__(self, hostname, username=None, password=None,
                                                  'disable_known_hosts',
                                                  key_type=bool,
                                                  default=False)
+        self.env_vars = _get_env_vars(env_keep)
```

Comment:
OK, now I see why you deal with `None` in the function... It's certainly a matter of style, and mine is different. Just for the note, I would do something like:

```
if env_keep is None:
   self.env_vars = {}
...
```

But it's up to you... again, just a matter of style.